src/gui/log/loglistview.cpp

```diff
@@ -79,24 +99,23 @@ namespace
                 font.setPointSizeF(option.font.pointSizeF());
             painter->setFont(font);
 
-            const QPen originalPen = painter->pen();
-            QPen coloredPen = originalPen;
-            coloredPen.setColor(index.data(BaseLogModel::TimeForegroundRole).value<QColor>());
-            painter->setPen(coloredPen);
+            QPalette palette = option.palette;
+
             const QString time = index.data(BaseLogModel::TimeRole).toString();
-            style->drawItemText(painter, textRect, option.displayAlignment, option.palette, (option.state & QStyle::State_Enabled), time);
+            palette.setColor(QPalette::Active, QPalette::WindowText, index.data(BaseLogModel::TimeForegroundRole).value<QColor>());
```

Comment:
Instead of changing `coloredPen` I choose to modify `palette` since it has less function calls and looks more logical to me.